htdocs/templates2/ocstyle/viewcache.tpl

```diff
@@ -72,6 +72,12 @@
         document.getElementById('convert_waypoints').style.display = 'none';
     }
 
+    function showalllists()
```

Comment:
I know it's messy already. But I would like to stick to one of the already used ways on how to write names. So `showAllLists` or `show_all_lists`. Maybe we need to agree on a style project wiede :-)